packages/editor-ui/src/views/WorkflowsView.vue

```diff
@@ -48,9 +59,16 @@
 					@click="goToTemplates"
 					data-test-id="new-workflow-template-card"
 				>
-					<n8n-icon :class="$style.emptyStateCardIcon" icon="box-open" />
+					<n8n-icon
+						:class="$style.emptyStateCardIcon"
+						:icon="isDemoTest ? 'graduation-cap' : 'box-open'"
+					/>
 					<n8n-text size="large" class="mt-xs" color="text-base">
-						{{ $locale.baseText('workflows.empty.browseTemplates') }}
+						{{
+							isDemoTest
+								? $locale.baseText('workflows.empty.viewDemo')
+								: $locale.baseText('workflows.empty.browseTemplates')
```

Comment:
Nit: We could save some lines here by using tertiary condition
```
$locale.baseText(`workflows.empty.${isDemoTest ? 'viewDemo' : 'browseTemplates'}`)
```

Reply:
nice. I like translation keys to be as searchable so I try to avoid this.. but I updated it so that it's one function